### ISSUE TITLE
Can't parse multiple flexible nowdoc blocks with different indentation

### ISSUE THREAD

**Issue by czosel** [glayzzle/php-parser#508]
Input:
```php
<?php
$a = <<<'EOT'
    a

    EOT;

$b = <<<'EOT'
  b
  EOT;
```
Output: `SyntaxError: Invalid body indentation level (expecting an indentation at least 2) on line 5`

**Comment by ichiriac** [comment]
Hi @czosel,

#508 & #509 seems to be fixed by PR #506 - you can still add new tests to check or close this issue and PR

**Comment by czosel** [comment]
:tada: Awesome! I'll go ahead and close this then :slightly_smiling_face:
